bin/syclcc-clang

```diff
@@ -271,6 +273,9 @@ class syclcc_config:
         flag.commandline, flag.environment
     ))
 
+  def _get_cpu_substitution_vars(self):
+    return {}
```

Comment:
I feel we should at least expose `$HIPSYCL_PATH` and `$HIPSYCL_LIB_PATH` in analogy to the cuda and rocm substitution vars. This can be convenient for e.g. manually rpath-ing the hipSYCL library path.